packages/bot/src/constants/bot-contents.js

```diff
@@ -0,0 +1,4 @@
+export const tabs_title = {
+    CHART    : 'Chart',
+    WORKSPACE: 'Workspace',
```

Comment:
Should this be `localize`d?

Reply:
@aaron-binary no i want to use it as enum, i did localized it when i want to use it as a title. please check line 33 of `footer-extension.js`